tests/endpoints_test.rs

```diff
@@ -139,4 +146,50 @@ mod tests {
 
         assert_eq!(66, local_body_bytes.len());
     }
+
+    // Test: `icebreakers register` success registration
+    #[tokio::test]
+    async fn test_icebreakers_registrations() -> Result<(), Box<dyn std::error::Error>> {
+        initialize_logging();
+
+        let (app, _, icebreakers_api, api_prefix) = build_app_non_solitary()
+            .await
+            .expect("Failed to build the application");
+
+        let ip_addr: IpAddr = "0.0.0.0".parse().unwrap();
+        let address = SocketAddr::new(ip_addr, 3000);
```

Comment:
Since these are tests, we could bind to port 0 (i.e. random), and then get the actual value back from the listener (`listener.local_addr().port` or similar).

Reply:
Ah, no, wait you _need_ it to be 3000, because that’s what you have open on `iptables`!